app/components/UI/Earn/components/ResourceToggle/ResourceToggle.styles.ts

```diff
@@ -0,0 +1,53 @@
+import { StyleSheet, ViewStyle } from 'react-native';
+import type { Theme } from '../../../../../util/theme/models';
+import { BackgroundColor } from '../../../Box/box.types';
+
+const styleSheet = (params: { theme: Theme; vars?: { style?: ViewStyle } }) => {
+  const { theme, vars } = params;
+  const { colors } = theme;
+
+  return StyleSheet.create({
+    container: Object.assign(
+      {
+        paddingHorizontal: 16,
+        paddingTop: 12,
+        marginBottom: 8,
+      },
+      vars?.style,
+    ),
+    group: {
+      position: 'relative',
+      width: '100%',
+      alignSelf: 'stretch',
+      backgroundColor: colors.background.alternative,
```

Comment:
```suggestion
      backgroundColor: colors.background.default,
```
This will ensure alignment with the original design specs:

<img width="994" height="782" alt="Screenshot 2025-10-25 at 1 44 04 AM" src="https://github.com/user-attachments/assets/2db956b3-888e-40d1-87ca-aaba53988fd4" />